sub-logs/studiodev.json

```diff
@@ -0,0 +1,19 @@
+{
+    "owner": {
+        "username": "ashfaq1994",
+        "email": "tricksfree.in@gmail.com"
+    },
+
+    "target": {
+        "A": {
+            "name": "studiodev",
+            "value": ["103.191.209.47"]
+        },
+        "A": {
+            "name": "www",
```

Comment:
A record is invalid, we do not support nested domains. You will have to do without www.